# Silesian translation for apport
# Copyright (c) 2019 Rosetta Contributors and Canonical Ltd 2019
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-12-07 16:23+0100\n"
"PO-Revision-Date: 2019-06-04 16:41+0000\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: Silesian <szl@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Zebranie informacyji ô systymie"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr "Żeby zebrać informacyje ô systymie, potrzebno je autoryzacyjo"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Reporty problymōw systymowych"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr "Wkludź hasło, żeby dostać dostymp do reportōw ô błyndach systymowych"

#: ../apport/ui.py:165
msgid "This package does not seem to be installed correctly"
msgstr "Tyn paket niy wyglōndo na zainstalowany noleżnie"

#: ../apport/ui.py:175
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:205
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Niykere wersyje paketōw sōm już stare. Zaktualizuj te pakety i wejzdrzij, "
"eli problym durch wystympuje:\n"
"\n"
"%s"

#: ../apport/ui.py:359
msgid "unknown program"
msgstr "niyznōmy program"

#: ../apport/ui.py:362
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Program %s niyspodzianie sie zawar"

#: ../apport/ui.py:365 ../apport/ui.py:1899
#, python-format
msgid "Problem in %s"
msgstr "Problym we %s"

#: ../apport/ui.py:370
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"We kōmputrze niy styko wolnyj pamiyńci, coby autōmatycznie przeanalizować "
"problym i zgłosić feler do twōrcōw programu."

#: ../apport/ui.py:430 ../apport/ui.py:448 ../apport/ui.py:621
#: ../apport/ui.py:630 ../apport/ui.py:867 ../apport/ui.py:1677
#: ../apport/ui.py:1860 ../apport/ui.py:1865
msgid "Invalid problem report"
msgstr "Niynoleżne zgłoszynie ô błyndzie"

#: ../apport/ui.py:431
msgid "You are not allowed to access this problem report."
msgstr "Niy mosz dostympu do tego zgłoszynio ô błyndzie."

#: ../apport/ui.py:439
msgid "Error"
msgstr "Feler"

#: ../apport/ui.py:441
msgid "There is not enough disk space available to process this report."
msgstr "Na dysku niy styko wolnego placu, coby przetworzić to zgłoszynie."

#: ../apport/ui.py:479
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:481
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:492 ../apport/ui.py:597
msgid "Invalid PID"
msgstr "Niynoleżny idyntyfikatōr procesu"

#: ../apport/ui.py:493
msgid "The specified process ID does not exist."
msgstr "Ôkryślōny idyntyfikatōr procesu niy istnieje."

#: ../apport/ui.py:498
msgid "Not your PID"
msgstr "Niy twōj PID"

#: ../apport/ui.py:499
msgid "The specified process ID does not belong to you."
msgstr "Ôkryślōny idyntyfikatōr procesu niy noleży do ciebie."

#: ../apport/ui.py:554
msgid "No package specified"
msgstr "Niy ôkryślōno paketu"

#: ../apport/ui.py:556
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Trzeba ôkryślić paket abo idyntyfikatōr procesu. Użyj ôpcyje --help, coby "
"dostać wiyncyj informacyji."

#: ../apport/ui.py:584
msgid "Permission denied"
msgstr "Brak dostympu"

#: ../apport/ui.py:586
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Ôkryślōny proces niy noleży do ciebie. Ôtwōrz tyn program jako posiedziciel "
"procesu abo root."

#: ../apport/ui.py:599
msgid "The specified process ID does not belong to a program."
msgstr "Ôkryślōny idyntyfikatōr procesu niy noleży do programu."

#: ../apport/ui.py:623
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Skrypt symptōmu %s niy ôkryślōł tykniyntego paketu"

#: ../apport/ui.py:631
#, python-format
msgid "Package %s does not exist"
msgstr "Paket %s niy istnieje"

#: ../apport/ui.py:660 ../apport/ui.py:872 ../apport/ui.py:905
#: ../apport/ui.py:915
msgid "Cannot create report"
msgstr "Niy idzie stworzić reportu"

#: ../apport/ui.py:675 ../apport/ui.py:734 ../apport/ui.py:752
msgid "Updating problem report"
msgstr "Uaktualnianie zgłoszynio ô błyńdzie"

#: ../apport/ui.py:677
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Niy je żeś autōr ani abōnynt tego zgłoszynio ô felerze abo zgłoszynie to je "
"tuplikat, abo było już zawarte.\n"
"\n"
"Zgłoś feler bez noczynie „apport-bug”."

#: ../apport/ui.py:689
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Niy je żeś autōr tego zgłoszynio ô felerze. Mocka prościj ôznaczyć feler za "
"tuplikat inkszego, aniżeli przeniyś swoje kōmyntorze i przidowki do nowego "
"feleru.\n"
"\n"
"Beztōż nojlepij wysłać nowy report ô felerze bez noczynie „apport-bug” i "
"przidać kōmyntorz do tego zgłoszynio ô już zgłoszōnym felerze.\n"
"\n"
"Na isto chcesz kōntynuować?"

#: ../apport/ui.py:735 ../apport/ui.py:753
msgid "No additional information collected."
msgstr "Żodne ekstra informacyje niy były zebrane."

#: ../apport/ui.py:816
msgid "What kind of problem do you want to report?"
msgstr "Jaki problym chcesz zgłosić?"

#: ../apport/ui.py:837
msgid "Unknown symptom"
msgstr "Niyznōmy symptōm"

#: ../apport/ui.py:838
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Symptōm „%s” je niyznōmy."

#: ../apport/ui.py:874
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:890
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Po zawrzyniu tyj wiadōmości kliknij ôkno programu, coby zgłosić jego problym."

#: ../apport/ui.py:907 ../apport/ui.py:916
msgid "xprop failed to determine process ID of the window"
msgstr "xprop niy poradziōł znojś idyntyfikatora procesu tego ôkna"

#: ../apport/ui.py:931
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:932
msgid "Specify package name."
msgstr "Ôkryśl miano paketu."

#: ../apport/ui.py:938 ../apport/ui.py:1070
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Przidowo ekstra etyketa do zgłoszynio. Idzie użyć pora razy."

#: ../apport/ui.py:976
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:987
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Zacznij w trybie zgłoszanio błyndu. Wymogo ôpcyje --package jak tyż "
"ôpcjōnalnie --pid, abo ino ôpcyje --pid. Jeźli żodno ôpcyjo niy ôstanie "
"użyto, pokoż wykoz symptōmōw."

#: ../apport/ui.py:996
msgid "Click a window as a target for filing a problem report."
msgstr "Kliknij w ôkno jako cyj do wysłanio zgłoszynio problymu."

#: ../apport/ui.py:1005
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Zacznij w trybie aktualizacyje zgłoszynio błyndu. Idzie użyć z ôpcyjōm --"
"package."

#: ../apport/ui.py:1014
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Zgłoś błōnd ô symptōmie (wymuszōne, jeźli było podano miano symptōmu za "
"parameter)."

#: ../apport/ui.py:1023
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Ôkryśl miano paketu w trybie --file-bug. Ôpcjōnalne, jeźli użyto była ôpcyjo "
"--pid (wymuszōne, jeźli było podano miano paketu za jedyny parameter)."

#: ../apport/ui.py:1034
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Ôkryśl program w trybie --file-bug. Jak to je ôbrane, to we zgłoszyniu "
"bydzie wiyncyj informacyji  (wymuszōne, jak pid je podany za jedyny "
"parameter)."

#: ../apport/ui.py:1042
msgid "The provided pid is a hanging application."
msgstr "Podany pid to numer zawieszōnyj aplikacyje."

#: ../apport/ui.py:1050
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Zgłoś błōnd fungowanio programu podle ôkryślōnego zbioru .apport abo .crash "
"zamiast tych, co czekajōm w %s (wymuszōne, jeźli zbiōr bōł dany za jedyny "
"parameter)."

#: ../apport/ui.py:1060
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"Zapisuje informacyje ô błyndzie do zbioru zamiast zgłoszać go. Report "
"zawarty we zbiorze, może być zgłoszōny niyskorzij z inkszego kōmputra."

#: ../apport/ui.py:1078
msgid "Print the Apport version number."
msgstr "Pokoż informacyje ô wersyji Apport."

#: ../apport/ui.py:1240
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "Analiza awaryje w terminalu ze pōmocōm apport-retrace."

#: ../apport/ui.py:1244
msgid "Run gdb session"
msgstr "Ôtwōrz sesyjo gdb"

#: ../apport/ui.py:1245
msgid "Run gdb session without downloading debug symbols"
msgstr "Ôtwōrz sesyjo gdb bez pobiyranio symbolōw debugowanio"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1247
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Aktualizacyjo %s ze pōmocōm fully symbolic stack trace"

#: ../apport/ui.py:1309
msgid "Can't remember send report status settings"
msgstr "Niy pamiyntōm sztelōnkōw sztandu wysyłanio reportōw"

#: ../apport/ui.py:1313
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""
"Spamiyntowanie sztandu reportowanio błyndu sie niy podarziło. Niy szło "
"nasztalować trybu auto abo trybu niy reportowanio nigdy."

#: ../apport/ui.py:1396 ../apport/ui.py:1409
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "Zgłoszynie problymu tyko programu, co już niyma zainstalowany."

#: ../apport/ui.py:1433
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "Problym tyko programu %s, co bōł zmiyniōny ôd czasu awaryje."

#: ../apport/ui.py:1494 ../apport/ui.py:1618 ../apport/ui.py:1869
msgid "This problem report is damaged and cannot be processed."
msgstr "Te zgłoszynie ô problymie je poprzniōne i niy może być przetworzōne."

#: ../apport/ui.py:1503
msgid "This report is about a package that is not installed."
msgstr "To zgłoszynie tyko paketu, co niyma zainstalowany."

#: ../apport/ui.py:1511
msgid "An error occurred while attempting to process this problem report:"
msgstr "Trefiōł sie feler w czasie prōby zgłoszynio błyndu:"

#: ../apport/ui.py:1528
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1533
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1534
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1575
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1580
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1679
msgid "Could not determine the package or source package name."
msgstr "Niy idzie znojś paketu abo miana paketu zdrzōdłowego."

#: ../apport/ui.py:1704
msgid "Unable to start web browser"
msgstr "Niy idzie ôtworzić przeglōndarki internetowyj"

#: ../apport/ui.py:1705
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Niy idzie ôtworzić przeglōndarki internetowyj, żeby ôtworzić %s."

#: ../apport/ui.py:1815
msgid "Network problem"
msgstr "Problym z necym"

#: ../apport/ui.py:1819
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Niy idzie połōnczyć sie z bazōm danych awaryji, wejzdrzij na swoje "
"połōnczynie internetowe."

#: ../apport/ui.py:1851
msgid "Memory exhaustion"
msgstr "Pamiyńć spotrzebowano"

#: ../apport/ui.py:1853
msgid "Your system does not have enough memory to process this crash report."
msgstr "Systymowi niy styko pamiyńci, coby przetworzić tyn report błyndu."

#: ../apport/ui.py:1904
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Niy idzie zgłosić problymu:\n"
"\n"
"%s"

#: ../apport/ui.py:1960 ../apport/ui.py:1972
msgid "Problem already known"
msgstr "Problym je już znōmy"

#: ../apport/ui.py:1962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Tyn problym bōł już zgłoszōny w reporcie pokozanym w przeglōndarce "
"internetowyj. Wejzdrzij, eli idzie przidać jake dalsze informacyje, co mogōm "
"być pōmōc twōrcōm programu."

#: ../apport/ui.py:1974
msgid "This problem was already reported to developers. Thank you!"
msgstr "Programisty sōm już poinformowani ô tym problymie. Dziynkujymy!"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "Wciś leda kery knefel, żeby kōntynuować..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "Co chcesz zrobić? Możesz:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "Ôbier (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i bajtōw)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:406
msgid "(binary data)"
msgstr "(dane binarne)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:199
msgid "Send problem report to the developers?"
msgstr "Zgłosić problym twōrcōm programu?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Jak report bydzie wysłany, to wypołnij formular\n"
"we autōmatycznie ôtwartyj przeglōndarce internetowyj."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "Wyślij &report (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "&Zbadej lokalnie"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "&Pokoż report"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "Ôstow zbiōr reportu, żeby niys&korzij wysłać go abo skopiować"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Pociep i &ignoruj prziszłe awaryje tyj wersyje programu"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:332 ../bin/apport-cli.py:366
#: ../bin/apport-cli.py:389
msgid "&Cancel"
msgstr "&Pociep"

#: ../bin/apport-cli.py:273
msgid "Problem report file:"
msgstr "Zbiōr reportu problymu:"

#: ../bin/apport-cli.py:279 ../bin/apport-cli.py:284
msgid "&Confirm"
msgstr "&Potwiyrdź"

#: ../bin/apport-cli.py:283
#, python-format
msgid "Error: %s"
msgstr "Błōnd: %s"

#: ../bin/apport-cli.py:289 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "Zbiyranie informacyji ô błyndzie"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Zebrane informacyje mogōm być wysłane do twōrcōw, coby ulepszyć\n"
"program. To może potrwać pora minut."

#: ../bin/apport-cli.py:307 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:511
msgid "Uploading problem information"
msgstr "Wysyłanie informacyje ô problymie"

#: ../bin/apport-cli.py:309
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Zebrane informacyje sōm wysyłane do systymu zgłoszanio felerōw.\n"
"To może potrwać pora minut."

#: ../bin/apport-cli.py:365
msgid "&Done"
msgstr "&Gotowe"

#: ../bin/apport-cli.py:371
msgid "none"
msgstr "brak"

#: ../bin/apport-cli.py:373
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Zaznaczōno: %s. Wielokrotny ôbiōr:"

#: ../bin/apport-cli.py:390
msgid "Choices:"
msgstr "Ôbiōr:"

#: ../bin/apport-cli.py:404
msgid "Path to file (Enter to cancel):"
msgstr "Drōga do zbioru (ENTER, żeby pociepnōńć):"

#: ../bin/apport-cli.py:410
msgid "File does not exist."
msgstr "Zbiōr niy istnieje."

#: ../bin/apport-cli.py:412
msgid "This is a directory."
msgstr "To je katalog."

#: ../bin/apport-cli.py:418
msgid "To continue, you must visit the following URL:"
msgstr "Coby kōntynuować, trzeba ôtworzić ta adresa URL:"

#: ../bin/apport-cli.py:421
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Idzie teroz ôtworzić przeglōndarka internetowo, abo wrazić ta adresa URL do "
"przeglōndarki na inkszym kōmputrze."

#: ../bin/apport-cli.py:427
msgid "Launch a browser now"
msgstr "Ôtwōrz teroz przeziyroczka"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Niy ma zgłoszyń awaryje, co czekajōm. Użyj ôpcyje --help, coby dostać "
"wiyncyj informacyji."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Niy przidowej nowych szlakōw do zgłoszynio ino wypisuj je do stdout."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Ôtwōrz interaktywno sesyjo gdb na ściepniyńcie pamiyńci zgłoszynio (ignoruje "
"ôpcyjo -o; niy przepisuje zgłoszynio)"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Wpisz zmiyniōne zgłoszynie do danego zbioru zamiast zmiyniać ôryginalne "
"zgłoszynie"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Skasuj ściepniyńcie spamiyńci ze zgłoszynio po regyneracyji szlaku sztapla."

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "Zastōmp CoreFile reportu"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "Zastōmp ExecutablePath reportu"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "Zastōmp ProcMaps reportu"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "Przebuduj informacyje ô pakecie we zgłoszyniu"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Zuduj tymczasowo zandkastla i pobier/zainstaluj w nij przidajne pakety "
"społym z symbolami debugowanio. Bez tyj ôpcyje założōne ôstowo, iże "
"potrzebne pakety i symbole sōm już zainstalowane w systymie. Argumynt "
"skazuje na katalog bazy kōnfiguracyje systymu paketōw. Ôbranie ôpcyje "
"„systym” prziniesie użycie zbiorōw kōnfiguracyje systymu - możliwe bydzie "
"jyny śledzynie i analiza (retrace) awaryji, co sie trefiyły w teroźnie "
"zainstalowanym wydaniu systymu."

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"Zbuduj nastympno tymczasowo zandkastla do instalowanio gdb i jego zależności "
"bez użycie tego samego wydanio jak we reporcie zamiast wersyje gdb, co mosz "
"zainstalowano."

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"Pokazuj posek postympu pobiyranio/instalowanio w czasie instalowanio paketōw "
"do zandkastle"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""
"Przidanie informacyje ô czasie do logów dlo ôperacyje trybu wsadowego (batch "
"operation)"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""
"Stwōrz i użyj repozytoriōw ôsōb trzecich ze zdrzōdeł ôkryślōnych w reportach"

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Katalog podrynczny paketōw pobiyranych do zandkastle"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Katalog na niyrozpakowane pakety. Prziszłe użycie założy, iże kożdy z już "
"pobranych paketōw je tyż rozpakowany do tyj zandkastle."

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "Instaluje ekstra paket do zandkastle (idzie użyć wiyncyj jak roz)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Drōga do zbioru z danymi autoryzacyje baz danych awaryji. Ôpcyjo używano w "
"czasie podowanio idyntyfikatora awaryje, żeby przesłać prześledzōne na nowo "
"szlaki sztapla (ino jeźli niy były użyte -g, -o, ani -s)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Pokoż wyśledzone na nowo szlaki sztapla i pytej ô potwierdzynie przed "
"wysłaniym ich do bazy awaryji."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Drōga do stuplowanyj bazy danych sqlite (wychodnie: bez wyszukowanio "
"tuplikatōw)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "Niy dodowej StacktraceSource do tego reportu."

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "Niy idzie użyć -C, jak niy ma -S. Zastawianie."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "Wysłać to za przidowki? [y/n]"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "Katalog cylu istnieje i niyma prōzny."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Wejzdrzij na strōna podryncznika po wiyncyj informacyji."

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "podej miano zbioru logōw, co je gyneruje valgrind"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"użyj zaś wcześnij stworzōnego katalogu zandkastle (SDIR) abo, jak go niy ma, "
"stwōrz go"

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"niy twōrz ani niy używej katalogu zandkastle na ekstra symbole debugowanio, "
"ino polygej na już zainstalowanych symbolach."

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"użyj zaś wcześnij stworzōnego katalogu cache (CDIR) abo, jak go niy ma, "
"stwōrz go"

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"pokazuj posek postympu pobiyranio/instalowanio w czasie instalowanio paketōw "
"do zandkastle"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Błōnd: niy idzie wykōnać %s. Zastawianie."

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"Trefiyło sie to w czasie poprzednigo uspanio i niy przizwolyło systymowi na "
"noleżne ôbudzynie."

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"Trefiyło sie to w czasie poprzednij hibernacyje i niy przizwolyło systymowi "
"na noleżne ôbudzynie."

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Proces budzynio zawiesiōł sie bardzo blisko kōńca i bydzie wyglōndać, choćby "
"skōńczōł normalnie."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr "Systym może być niysztabilny, bez to może być potrzebny restart."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Zgłoś problym..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Zgłoś twōrcōm błyndy fungowanio"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Program %s niyspodzianie stanōł."

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Program %s niyspodzianie sie zawar."

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:214 ../kde/apport-kde.py:266
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Trefiōł sie wewnyntrzny błōnd %s."

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:207
msgid "Send"
msgstr "Wyślij"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:666 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr "Pokoż informacyje"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:258 ../kde/apport-kde.py:276
msgid "Continue"
msgstr "Dalij"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "Program %s przestoł ôdpowiadać."

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Program „%s” przestoł ôdpowiadać."

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:222
#, python-format
msgid "Package: %s"
msgstr "Paket: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:229
msgid "Sorry, a problem occurred while installing software."
msgstr "Trefiōł sie problym przi instalowaniu ôprogramowanio."

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:238
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "Trefiōł sie wewnyntrzny błōnd programu %s."

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:245
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Program %s niyspodzianie sie zawar."

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:271
msgid "If you notice further problems, try restarting the computer."
msgstr "Jeźli pokożōm sie dalsze problymy, to sprōbuj zrestartować kōmputer."

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr "Ignorowanie problymōw tyj zorty w prziszłości"

#: ../gtk/apport-gtk.py:670 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr "Skryj informacyje"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:552 ../kde/apport-kde.py:595
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Pociep"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Zgłoszynie awaryje"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Trefiōł sie wewnyntrzny błōnd.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "Pamiyntej w prziszłości"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Ignorowanie prziszłych błyndōw tyj wersyje programu"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "Ôtwōrz ta aplikacyjo na nowo"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Zbadej lokalnie"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "Niy wysyłej"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Zbiyranie informacyji ô błyńdzie</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Trwo zbiyranie informacyji,co mogōm pōmōc twōrcōm programu sprawić zgłoszōny "
"problym."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Wysyłanie informacyje ô błyńdzie</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:513
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Zebrane informacyje sōm wysyłane do systymu śledzynio felerōw. To może "
"potrwać pora minut."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Zbiōr zgłoszynio Apport"

#: ../kde/apport-kde.py:261
msgid "Leave Closed"
msgstr "Ôstow zawarty"

#: ../kde/apport-kde.py:262 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr "Ôtwōrz na nowo"

#: ../kde/apport-kde.py:355
msgid "Username:"
msgstr "Miano ôd używocza:"

#: ../kde/apport-kde.py:356
msgid "Password:"
msgstr "Hasło:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr "Zbiyranie informacyji ô problymie"

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Zebrane informacyje mogōm być wysłane do twōrcōw, coby ulepszyć program. To "
"może potrwać pora minut."

#: ../kde/apport-kde.py:510
msgid "Uploading Problem Information"
msgstr "Wysyłanie informacyje ô problymie"
